### ISSUE TITLE
Complete image task 

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#1782]
This is related to issue #1717, we need to resolve the errors discovered in review process (alt text, tagging, missing images, etc). and compete task with instructions given.

**Comment by mcharg** [comment]
Here's what I'm doing for this task.

1. Created a list of all the images on the site using the prototype, live site, and dev site. Noted which are duplicates. Put this list in a new tab in the Content Tracking spreadsheet.
2. Going to each image in the list on the dev site and inspecting to see which image is actually being used on the page.
3. Marking the image with a Correct tag if it's using the expected image or fixing the reference to the image in the page if it isn't. 
4. Decided on the best image and alt text to use for duplicates.
5. Added missing images.
5. Making sure each image is properly tagged.
6. Making sure each image has alt text.
7. Noting images that need better alt text in spreadsheet.
8. Copying the image url into the spreadsheet.
9. Marking the image as ready to review in the spreadsheet.

I got through everything except the valuation examples this afternoon. Will finish tomorrow.

**Comment by mcharg** [comment]
@cthomasONRR Images are ready for you to review.

1. All the images are in the Images tab in the Content Tracking sheet. Those ready to review have a status of "Ready to Review."
1. I used "Correct" tags to keep track of ones I reviewed. I deleted them after I went through and deleted any images without them that weren't being used. If there are any lingering "Correct" tags, they can be removed.
2. I didn't look at the alt text in detail, but marked whether it exists in the spreadsheet. Only one image is missing alt text.  I also marked any that looked like they weren't very detailed with "Review Needed" in the spreadsheet.
3. I tried to be consistent with the labels, but it would be great to have a second set of eyes on those.
3. There's one ONRR logo image with no tags. I left it there because I can't tell if it's being used in the header/footer. I also couldn't figure out where the image in the Revenue Statistics section of the homepage is added, so I didn't know what's being used for that. @jlouis-onrr Can you help identify where the images in the header/footer and homepage are?
3. There's one image left in the File Library folder that it won't let me delete.

**Comment by cthomasONRR** [comment]
@mcharg Ok I completed reviewing all images that had the "Ready to Review" status. I saw no major issues. 

- I renamed lots of titles for clarity and made sure the titles matched within the CMS and the ContentTracking spreadsheet
- I added several descriptor tags, but the main tags (levels 1-4 navigation) were all there and correct
- I edited a few alt texts where I saw obvious typos (I know this wasn't part of this issue, but just edited obvious errors, did not create alt text)
- I agreed with all the "Review Needed" for AltText. I will address these in issue #1786
